akommons/src/main/java/com/dewarder/akommons/content/res/TypedArray.kt

```diff
@@ -37,4 +43,16 @@ inline fun <R> TypedArray.use(block: (TypedArray) -> R): R {
             recycle()
         }
     }
-}
\ No newline at end of file
+}
+
+inline fun <R> Context.useStyledAttributes(@StyleableRes attrs: IntArray, block: (TypedArray) -> R): R
```

Comment:
It will be better to move all extensions with receiver of `Context` to file with same name which is placed in `akommons.content.Context`